### ISSUE TITLE
Filter for start date allows end before start date

### ISSUE THREAD

**Issue by marcusm7** [opencast/opencast-admin-interface#920]
(The same day for start and end bug which returned an empty list has been fixed - works for me. Thanks. :)
The filter for start date  allows end before start date.

Steps:
- open admin ui
- select filter for start date
- choose today for start 
- choose yesterday for end

This shouldn't be possible.

System:
PC, Linux, Firefox 130.0.1 (64-bit), Opencast 16.x – b0e0131 – built on 2024-09-29

![image](https://github.com/user-attachments/assets/47973956-f9ce-456b-bac2-737637a4fe0c)

Suggestion: A date range picker which allows to pick start and end in the same picker (to safe the user some clicks).
![image](https://github.com/user-attachments/assets/ab680a82-7620-4056-ae31-9388a0464269)  
  
related: https://github.com/opencast/opencast-admin-interface/issues/614 addresses date picker issues, too.

**Comment by snoesberger** [comment]
I agree with @marcusm7, a date range picker would be much more user friendly (see also this issue #948 ).
Looking at the source code, I can see that the [MUI DatePicker](https://mui.com/x/react-date-pickers/date-picker/) is currently used. There is also a [MUI DateRangePicker](https://mui.com/x/react-date-pickers/date-range-picker/), but you have to pay for it. From my point of view it would therefore make sense to replace the MUI DatePicker with something free that also allows date range picking. And to have just one dependency and a consistent design, replace the date picker also in other places in the admin UI (and at the same time address #614).

**Comment by Arnei** [comment]
As far as I can tell this issue has been resolved by #986, so I'll close it. If it has not been resolved, please reopen it.
